src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/config/UserContext.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.config
+
+import org.springframework.stereotype.Component
+
+@Component
+object UserContext {
+  private val authToken = ThreadLocal<String>()
+  fun setAuthToken(aToken: String) {
+    authToken.set(aToken)
+  }
+  fun getAuthToken(): String {
+    return authToken.get()
```

Comment:
Don't think you need getters and setters?

Reply:
In this case they're not just getters and setters.. they are returning a value inside a ThreadLocal object